src/main/java/ai/knowly/langtorch/processor/minimax/MiniMaxServiceProvider.java

```diff
@@ -0,0 +1,38 @@
+package ai.knowly.langtorch.processor.module.minimax;
+
+import ai.knowly.langtorch.llm.minimax.MiniMaxApi;
+import ai.knowly.langtorch.llm.minimax.MiniMaxService;
+import ai.knowly.langtorch.utils.Environment;
+import ai.knowly.langtorch.utils.api.key.MiniMaxKeyUtil;
+import com.google.common.flogger.FluentLogger;
+import java.time.Duration;
+
+/**
+ * @author maxiao
+ * @date 2023/06/07
+ */
+public final class MiniMaxServiceProvider {
```

Comment:
Could you replace this with Guice module? 

example:
https://github.com/Knowly-ai/langtorch/blob/55f9b370a883e5a40bc20eaab51f1fee56fcdfb0/src/main/java/ai/knowly/langtorch/llm/openai/OpenAIServiceConfigWithExplicitAPIKeyModule.java#L8

Reply:
had done